### ISSUE TITLE
comments on blog post

### ISSUE THREAD

**Issue by sckott** [adamhsparks/GSODR_blog_post#1]
* this sentence (from 1st paragraph) wording seems off _Other colleagues that work with crop biophysical models to model yields and have similar weather data needs and issues with data availability._
* `Raster` is all lowercase
* change `Scott Chamberlin` to `Scott Chamberlain`
* `meta data` is one word i think
* In __Conclusions__ section: _but in the more recent years the data become more reliable and more stations are being added._  maybe `became` ?
* could you link to the review in onboarding

questions:

* what link do you want to have behind your name at top of the blog post?

**Comment by adamhsparks** [comment]
Thanks, @sckott.

I've made changes as suggested.  

* I've reworded the entire paragraph to read more clearly

* I've fixed "Raster" to read "raster"

* I've corrected the spelling of your name. (oops)

* I've made `meta data` one word

* I've replaced `become` with `became`

* I have a link to the review in the acknowledgments, however, I didn't have it formatted properly, I've fixed that

* I've also addressed some other formatting and grammatical errors that I found that were not mentioned in this issue

For the link after my name, let's use my official uni profile:  https://staffprofile.usq.edu.au/Profile/Adam-Sparks

I've included my affiliation after my name as well, if we can use that?
